assets/src/hooks/useDetour.ts

```diff
@@ -153,8 +153,8 @@ export const useDetour = ({ routePatternId, shape }: OriginalRoute) => {
      * Sets {@link startPoint} if unset.
      * Otherwise sets {@link endPoint} if unset.
      */
-    addConnectionPoint:
-      state === DetourState.Finished ? undefined : addConnectionPoint,
+    addConnectionPoint,
+    //     state === DetourState.Finished ? undefined : addConnectionPoint,
```

Comment:
I _think_ this was dead code. `addConnectionPoint` is only callable when the original route shape is clickable, which isn't true in `DetourState.Finished`.